### ISSUE TITLE
TypeError: tuple indices must be integers or slices, not tuple

### ISSUE THREAD

**Issue by bn2022** [bowu1004/instance_segmentation_RealSense#8]
File ".../instance_segmentation_RealSense/sort.py", line 199, in associate_detections_to_trackers
    if (d not in matched_indices[:, 0]):
TypeError: tuple indices must be integers or slices, not tuple

hi, I am getting this error do you know the reason?

**Comment by bowu1004** [comment]
I'm not sure about this but hopefully this will help at https://stackoverflow.com/questions/39839034/python-tuple-indices-must-be-integers-not-tuple.

**Comment by bn2022** [comment]
yes I saw this page and converted matched_indices to np.array(matched_indices)
but got another error.
You did not get these errors?

**Comment by bowu1004** [comment]
Sorry, no. But I only run in macOS and Win10, not Ubuntu.
